CHANGELOG.md

```diff
@@ -47,4 +47,17 @@ The format is based on [Keep a Changelog 1.0.0].
 ## [Release 0.1.4]
 - Initial tagged release
 
+[Unreleased]: https://github.com/nationalarchives/ds-caselaw-custom-api-client/compare/v0.4.5...HEAD
+[Release 0.4.5]: https://github.com/nationalarchives/ds-caselaw-custom-api-client/compare/v0.4.4...v0.4.5
```

Comment:
Should these be link to this repo (`ds-caselaw-utils` rather than `ds-caselaw-custom-api-client`)?